universityofadelaide/shepherd
# WEBOPS-229 Fix scaffolding

Removes files that are provided by scaffolding and in .gitignore. Shrinks composer.json making it in line with upstream changes to shepherd-drupal-scaffold and shepherd-drupal-project.

To test this PR:

```
cd <your code dir>
git clone git@github.com:universityofadelaide/shepherd.git shepherd-test
cd shepherd-test
git checkout feature/webops-229-fix-scaffolding
composer install
./dsh
robo build
```

This should leave you with a functioning Shepherd.

To test the wider goal of the WEBOPS-229 story:

```
git clone git@github.com:universityofadelaide/shepherd-drupal-project.git create-project-test
cd create-project-test
git checkout consolidate-scaffolding
composer install
./dsh
robo build
```

This should leave you with a functioning Drupal install with the deps required to run on the Shepherd stack, ready to run `git init` and make your first commit.

On acceptance:
* In shepherd-drupal-scaffold repo, merge `consolidate-scaffold` branch to `master`.
* Change version ref of shepherd-drupal-scaffold back to `dev-master` in both this repo and shepherd-drupal-project feature branches.
* Merge both feature branches in shepherd-drupal-scaffold and this repo.
